packages/squiggle-lang/src/rescript/Reducer/Reducer_MathJs/Reducer_MathJs_Eval.res

```diff
@@ -0,0 +1,33 @@
+module CTV = Reducer_Extension.CodeTreeValue
```

Comment:
We consult mathjs.evalauate when there are not functions found. Thus, mathjs library is available by default

Reply:
Yea, I get that, but I have mixed feelings about the dependency directions in this current directory structure. I'm curious if this could be neatened up.

Reply:
MathJs parser will be gone and we won't have that :) . It really works this way. Remember. Unit Tested.
RESOLVED